scripts/ensure-ecr-repository.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+ensure_repository() {
+    local aws_service=$1
+    local ecr_repo=$2
+    local region="us-east-1"
```

Comment:
I don't think this needs to be a variable. You can probably just hardcode `--region=us-east-1` into all of the `aws ecr-public` commands.